inc/namespace.php

```diff
@@ -26,7 +26,6 @@ function redirect_user() {
 			header( 'Access-Control-Allow-Credentials: true' );
 			header( 'Access-Control-Allow-Headers: Authorization, Content-Type' );
 			header( 'Access-Control-Allow-Expose-Headers: X-WP-Total, X-WP-TotalPages' );
-			exit;
```

Comment:
I don't think this is enough, as it'll fail the `$page` check below. Also, if we're sending the full OPTIONS response, we don't need to send these manual headers

Reply:
Yeah I'm not sure that will do what is intended. Can we just live this block below the early returns on logged in / allowed pages checks?

Reply:
Yeah actually, I think this whole block just wants to live down after the cap check, and that'll sort it

Reply:
It's working in the sense I can add audiences again but yeah makes sense - I'll put this in the block after the logged in check that handles REST requests

Reply:
Yeee that's what made me worry about putting specific success criteria :D